docs/docker-machine-drivers.md

```diff
@@ -55,6 +55,18 @@ $ newgrp libvirt
 <a name="xhyve-driver"></a>
 ## xhyve driver
 
+Minishift is currently tested against `docker-machine-driver-xhyve` 0.3.1
```

Comment:
Probably "with version" ahead of `0.3.1` would make it more readable.

`Minishift is currently tested against 'docker-machine-driver-xhyve' with version 0.3.1`.